sdk/cognitiveservices/cognitiveservices-qnamaker-runtime/README.md

```diff
@@ -0,0 +1,116 @@
+## An isomorphic javascript sdk for - QnAMakerRuntimeClient
+
+This package contains an isomorphic SDK for QnAMakerRuntimeClient.
+
+### Notes
```

Comment:
This header can be skipped and contents below can be combined with the line above.

> This package contains an isomorphic SDK for interacting with QnAMaker such as training and asking questions.

Reply:
Same feedback to the readme contents on the other package in this PR